src/Components/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,35 @@
+import { PortalWithState } from 'react-portal';
+import PropTypes from 'prop-types';
+import FocusTrap from 'focus-trap-react';
+import './Dialog.css';
+
+
+export function Dialog({ title, children, onClose }) {
+  return (<PortalWithState closeOnOutsideClick closeOnEsc defaultOpen onClose={() => onClose && onClose()}>
```

Comment:
We can use optional chaining operator here.
Before:
`onClose && onClose()`
After:
`onClose?.()`